src/cmd/nsfs.js

```diff
@@ -258,6 +258,7 @@ async function main(argv = minimist(process.argv.slice(2))) {
         const https_port_sts = Number(argv.https_port_sts) || config.ENDPOINT_SSL_STS_PORT;
         const metrics_port = Number(argv.metrics_port) || config.EP_METRICS_SERVER_PORT;
         const forks = Number(argv.forks) || config.ENDPOINT_FORKS;
+        if (forks > 0) process.env.ENDPOINT_FORKS = forks.toString();
```

Comment:
As I mentioned on Slack, after the new configuration, this is not needed if setting it to config.json.

Reply:
used for argv.forks to work. resolving.